source/features/conversation-activity-filter.tsx

```diff
@@ -147,17 +146,25 @@ function addWidget(position: Element): void {
 	);
 }
 
-function init(): void {
-	const observer = observe(':is(.gh-header-sticky .meta, .gh-header-meta .flex-auto):not(.rgh-conversation-activity-filter)', {
-		add: addWidget
-	});
-	deinit.push(observer.abort);
+async function init(): Promise<void> {
+	const mainHeader = await elementReady('.gh-header-meta .flex-auto:not([id])');
+	const stickyHeader = await elementReady('.gh-header-sticky .meta');
+	for (const header of [mainHeader, stickyHeader]) {
+		if (select.exists('.rgh-conversation-activity-filter', header)) {
+			continue;
+		}
+
+		addWidget(header!);
+	}
```

Comment:
More like

```js
function init() {
  await addWidget(selector 1)
  await addWidget(selector 2)
}

Reply:
I aint done yet 😠. 

I had a better idea.

Reply:
How's

```js
 	void addWidget(elementReady(selector 1);
 	void addWidget(elementReady(selector 2);
```

better than

```js
 	await addWidget(selector 1);
 	await addWidget(selector 2);
```

?

Also `await` is advised because that's how we correctly catch errors and link them to the feature name

Reply:
ahh did not think of that